.htaccess

```diff
@@ -0,0 +1 @@
+Deny from all
```

Comment:
Do we need this?

Reply:
Good question. Something automatically added it, and since it's not the web directory I didn't suppose it hurt. Still not sure where it came from though. I'm happy with either removing or leaving it.